### ISSUE TITLE
Automate man page creation

### ISSUE THREAD

**Issue by Canop** [Canop/broot#418]
broot's man page should be easily kept synchronized with the code base. It seems clap doesn't have relevant features yet.

**Comment by adriansosa** [comment]
How do you feel about using `help2man`?

It can generate man pages in roff format from the `--help` and `--version` commands, while being able to override sections.

I suggest using `help2man` for generating the man page. This approach still keeps `clap_args.rs` as the source of truth, while giving you flexibility in editing the content in the man page. 

It also can generate `info` pages, that way we can meet the package requirements for Debian and other distros that require `info` pages (since GNU hates roff so much).

If so I would be more than happy to take it on.

**Comment by Canop** [comment]
The idea seems good. No opinion on the tool. Can you test and show how it would render and how it  would integrate in the build chain ? If necessary, come to the chat (during European hours).

**Comment by adriansosa** [comment]
Yeah I will put together some notes and a POC and join the chat during European hours.
